portal/src/main/webapp/js/src/oncoprint/webgl/dist/oncoprint-bundle.js

```diff
@@ -1656,14 +1657,17 @@ var OncoprintLabelView = (function () {
 		} else {
 		    var hovered_track = isMouseOnLabel(view, evt.pageY - view.$canvas.offset().top);
 		    if (hovered_track !== null) {
-			var $tooltip_div = $('<div>');
+			var $tooltip_div = $('<section>');
```

Comment:
oh, I see. but the difference is totally semantic, functionally its the same as a div? if so, then works for me